external/radiation/radiation/radiation_driver.py

```diff
@@ -328,9 +333,6 @@ def radupdate(
 
             return soldict, aerdict, gasdict
 
-        else:
-            return slag, sdec, cdec, solcon
```

Comment:
these outputs are not used anywhere?

Reply:
They aren't; the Fortran code uses this for a print statement and so it got ported that way. But the real way these values are used by having the `driver.sol` class store them internally.